buildkite/scripts/build-hardfork-package.sh

```diff
@@ -2,7 +2,23 @@
 
 set -eo pipefail
 
-([ -z ${DUNE_PROFILE+x} ] || [ -z ${CONFIG_JSON_GZ_URL+x} ] || [ -z ${NETWORK_NAME+x} ] || [ -z ${MINA_DEB_CODENAME+x} ]) && echo "required env vars were not provided" && exit 1
+# TODO: remove DUNE_PROFILE and configure using NETWORK_NAME
```

Comment:
Is this TODO still to be here or could be removed?

Reply:
Woops, I thought I did already. Fixing.